### ISSUE TITLE
!Rls : event not found

### ISSUE THREAD

**Issue by ApinkLoyalFan** [taengstagram/instagram-livestream-downloader#3]
## Please read the following instructions carefully

- Issues submitted without this template format will be **ignored**.
- I can't help you with python / ffmpeg installation problems so issues related to those will probably not get a response from me.
![git](https://cloud.githubusercontent.com/assets/27340917/24949413/b75f22ae-1f97-11e7-994e-d9cb3b7601a3.JPG)

- Please read the questions asked and answer completely.
- Put an `x` into all the boxes [ ] relevant to your issue (like so [x] - no spaces).
- Do not post screenshots of text errors or logs.
- Use the Preview tab to see how your issue will actually look like.

---

### Before submitting an issue, make sure you have:
- [x] Read the [README](https://github.com/taengstagram/instagram-livestream-downloader/blob/master/README.md)
- [x] Read the [FAQ](https://github.com/taengstagram/instagram-livestream-downloader/blob/master/FAQ.md)
- [x] [Searched](https://github.com/taengstagram/instagram-livestream-downloader/search?type=Issues) the bugtracker for similar issues including **closed** ones
- [x] Make sure you have the latest version: v0.2.8

### Purpose of your issue:
- [x] Bug report (problems/errors)
- [ ] Question
- [ ] Other

### For a bug report, you **must** include the command used and a complete output/error log.

Command:

```bash
# Paste your command here.
# Remember not to include sensitive info such as passwords. Example:
livestream_dl -u "myusername" -collectcomments -o "/path/to/dl_folder/" "myfavigacct"
```

Complete Output/Error Log:

```
# Paste the error/output log here
```

### Describe your issue
Hello there, sorry just have time to post issue.

I am success install the downloader but when I tried it it's not work.

I try input this livestream_dl -u girlsgenerationdevotee -p Password ayana_ksymova
and livestream_dl -u "girlsgenerationdevotee" -p "Password" "ayana_ksymova"
Note : I choose someone who still doing IG live randomnly
both resulting this !Rls : event not found.

Can you tell me what mistake I made ?

Thank you

Explanation of your issue goes here. Please make sure you describe the issue clearly and completely.

**Comment by taengstagram** [comment]
1. Are you using cygwin? If you are, try using single quotes ``'`` instead of double quotes ``"``.
2. If not, do you have an exclamation mark ``!`` in your password? Trying escaping the ``!`` by adding a backslash ``\`` before it, so that it becomes ``\!``.

```bash
livestream_dl -u "your.username" -p "\!yourpassword"
```

Also, please don't post screenshots of text errors in the future. You can just copy paste the text.

**Comment by ApinkLoyalFan** [comment]
Hello, thank you for your response.
I change my password into very simple one and got this 
INSTAGRAM LIVESTREAM DOWNLOADER (v0.2.8) [python=3.6.1,win32]
=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-
Traceback (most recent call last):
  File "e:\programs\python\lib\site-packages\instagram_private_api\client.py", line 439, in _call_api
    response = self.opener.open(req, timeout=self.timeout)
  File "e:\programs\python\lib\urllib\request.py", line 532, in open
    response = meth(req, response)
  File "e:\programs\python\lib\urllib\request.py", line 642, in http_response
    'http', request, response, code, msg, hdrs)
  File "e:\programs\python\lib\urllib\request.py", line 570, in error
    return self._call_chain(*args)
  File "e:\programs\python\lib\urllib\request.py", line 504, in _call_chain
    result = func(*args)
  File "e:\programs\python\lib\urllib\request.py", line 650, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 400: Bad Request

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "E:\Programs\Python\Scripts\livestream_dl-script.py", line 11, in <module>
    load_entry_point('instagram-livestream-downloader==0.2.8', 'console_scripts', 'livestream_dl')()
  File "e:\programs\python\lib\site-packages\livestream_dl\__main__.py", line 5, in main
    run()
  File "e:\programs\python\lib\site-packages\livestream_dl\download.py", line 318, in run
    res = api.username_info(argparser.instagram_user)
  File "e:\programs\python\lib\site-packages\instagram_private_api\endpoints\users.py", line 26, in username_info
    res = self._call_api('users/%(user_name)s/usernameinfo/' % {'user_name': user_name})
  File "e:\programs\python\lib\site-packages\instagram_private_api\client.py", line 449, in _call_api
    error_response=json.dumps(error_obj))
instagram_private_api.errors.ClientLoginRequiredError: login_required

I use git bash and I don't have ! in my password.

Thank you

**Comment by taengstagram** [comment]
In the folder that git bash starts from (probably your home folder), there should be a ``<your.username>.json`` file. Delete that file and run the command again.

I don't know how git-bash works but this may work for you:
```bash
# Show the .json files in current directory
ls *.json
# Delete girlsgenerationdevotee.json, assuming your username is girlsgenerationdevotee
rm girlsgenerationdevotee.json
# Try again
livestream_dl -u 'girlsgenerationdevotee' -p 'your.password' 'the.ig.streamer'
```

**Comment by ApinkLoyalFan** [comment]
Hello, I delete girlsgenerationdevotee.json and suddly it's work thank you
Btw if I don't set output folder where it will be saved ?
And is the password must be only letter or I can combine with number ?

Thank you

**Comment by taengstagram** [comment]
The default output folder is a ``downloaded`` folder, probably in your home folder.

Passwords of any kind should actually be fine. I have tested a password with ``!`` and it wasn't a problem. I gave you 2 suggestions before (switch to single quote or to use the ``\`` escape character) to try but you did not say if it worked or not.

If you change your password, you probably need to remove the ``<your.username>.json`` file again. I'll fix this in the future, but in the meantime, you'll need to manually delete.
